README.md

```diff
@@ -136,9 +137,11 @@ atomic access to their respective inner element(s), [Array] and
 into the inner element(s) through their respective inner [Miniconf] implementations.
 
 ## Formats
-The path hierarchy separator is the slash `/`.
 
-Values are serialized into and deserialized from JSON.
+Miniconf is generic over the `serde` backend/payload format and the path hierarchy separator
+(as long as the path can be split by it unambiguously).
+
+Currently support for `/` as the path hierarchy separator and JSON (`serde_json_core`) is implemented.
```

Comment:
I'd really like to see docs improved here. Maybe:
1. Show the user what an example of the default `json + slash` syntax looks like
2. Describe how the user could implement their own wire + path format.

Reply:
The example above comprehensively shows what that syntax looks like.
I think the best example on how to implement a `SerDe<Spec>` (that'll be maintained and tested) is the one we have. So we should just point at that.

Reply:
I think we need to add at least _some_ documentation to the `SerDe` trait to indicate what the intent is. Otherwise, I doubt any one would ever use the "custom serializer + path" syntax

Reply:
Added docs to `SerDe`